crates/bws/src/main.rs

```diff
@@ -152,12 +119,12 @@ async fn process_commands() -> Result<()> {
         | Commands::List {
             cmd: ListCommand::Projects,
         } => {
-            let projects = client
-                .projects()
-                .list(&ProjectsListRequest { organization_id })
-                .await?
-                .data;
-            serialize_response(projects, cli.output, color);
+            command::project::list(
+                client,
+                organization_id,
+                OutputSettings::new(cli.output, color),
```

Comment:
**suggestion**: Output settings are used multiple times, I think it would be nice to extract the creation to above and use the local instance in all the calls.

```rust
let output_settings = OutputSettings::new(cli.output, color);
```

Reply:
Updated in: https://github.com/bitwarden/sdk/pull/809/commits/826ba67ed9baa5103f575fdbc7f614846cd3ea54